File: dongyimai-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    this.selectOptionList = function () {
        return $http.get('../brand/selectOptionList.do');
    }

    this.save = function (methodStr,pojo) {
        return $http.post('../brand/'+methodStr+'.do',pojo);
    }

    this.search = function (page,rows,searchEntity) {
        return $http.post('../brand/search.do?page='+page+"&rows="+rows,searchEntity);
    }

    this.dele = function (ids) {
        return $http.get('../brand/dele.do?ids='+ids);
    }

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?bid='+id);
    }

    this.finaAll = function () {
        return $http.get('../brand/findAll.do');
    }

    this.findPage = function (page,rows) {
        return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
    }

});

File: dongyimai-sellergoods-interface/src/main/java/com/dongyimai/sellergoods/service/BrandService.java
package com.dongyimai.sellergoods.service;

import com.dongyimai.entity.PageResult;
import com.dongyimai.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    public List<TbBrand> findAll();

    public PageResult findPages(int pageNum,int pageSize);

    public void save(TbBrand tbBrand);

    public TbBrand findOne(Long bid);

    public void update(TbBrand tbBrand);

    public void delete(Long [] ids);

    public PageResult search(TbBrand tbBrand,int page,int rows);

    List<Map> selectOptionList();
}
